### PR TITLE
Improve imports

### PR DESCRIPTION
**Pull request by sbarzowski**
* Nonexistent file is now a runtime error (used to be ignored)
* Permission denied is a runtime error (used to be an internal error)
* Basic tests added for imports
* Renamed tests from .input to .jsonnet
	* It's standard extension for jsonnet code
	* It's compatible with C++ test suite
	* Editors enable syntax highlighting based on extension
	* Importing .input looks weird

### CODE REVIEW

**Comment by sbarzowski** [comment]
@lwander @sparkprime

**Comment by sbarzowski** [comment]
You mean the renaming of *.input to *.jsonnet? I've needed to import these files and it just felt weird to import "something.input". And I wanted to change it for some time now:
* It's standard extension for jsonnet code
* It's compatible with C++ test suite
* Editors enable syntax highlighting based on extension
* Importing .input looks weird

File: imports.go [hunk 1]
```diff
@@ -131,6 +130,7 @@ func tryPath(dir, importedPath string) (found bool, content []byte, foundHere st
 func (importer *FileImporter) Import(dir, importedPath string) *ImportedData {
 	found, content, foundHere, err := tryPath(dir, importedPath)
 	if err != nil {
+		fmt.Println(err)
```
Comment (sparkprime) [inline imports.go]: Looks like abandoned debug print

File: interpreter.go [hunk 2]
```diff
@@ -373,7 +373,7 @@ func (i *interpreter) evaluate(a ast.Node, context *TraceContext) (value, error)
 	case *ast.ImportStr:
 		// TODO(sbarzowski) put this information in AST instead of getting it out of tracing data...
```
Comment (sparkprime) [inline interpreter.go]: It is in the AST isn't it?  In the location field

**Comment by sparkprime** [comment]
@sbarzowski rebase needed

File: imports.go (hunk 1 above)
Comment (sbarzowski) [reply imports.go]: Right. Moreover the branch wasn't covered by tests so I didn't notice. 

Println removed, test added.

File: interpreter.go (hunk 2 above)
Comment (sbarzowski) [reply interpreter.go]: Uh, of course.

**Comment by sbarzowski** [comment]
@sparkprime rebased

**Comment by sbarzowski** [comment]
For the record, I needed to retry a build (through amending a commit message). 

It timed out on:
```
Updating gimme
$ GIMME_OUTPUT="$(gimme tip | tee -a $HOME/.bashrc)" && eval "$GIMME_OUTPUT"
```
which I believe installs Go.
(https://travis-ci.org/google/go-jsonnet/jobs/278737081)
(https://github.com/travis-ci/gimme)

**Pull request merged**